website/blog/2024-05-22-latest-dbt-stability-improvement-innovation.md

```diff
@@ -82,9 +82,9 @@ All incidents are retrospected to make sure we not only identify and fix the roo
 
 :::
 
-The outcome of this process is that, when you select "Versionless" in dbt Cloud, the time between an improvement being made to dbt Core and you *safely* getting access to it in your projects is a matter of days — rather than months of waiting for the next dbt Core release, on top of any additional time it may have taken to actually carry out the upgrade.
+The outcome of this process is that, when you select "Latest" in dbt Cloud, the time between an improvement being made to dbt Core and you *safely* getting access to it in your projects is a matter of days — rather than months of waiting for the next dbt Core release, on top of any additional time it may have taken to actually carry out the upgrade.
```

Comment:
```suggestion
The outcome of this process significant. When you select "Latest" in dbt Cloud, you gain faster access to improvements in dbt Core and safely getting access to it in your projects in a matter of days. As a opposed to waiting months for the next dbt Core release, on top of any additional time it may have taken to actually carry out the upgrade.
```